src/JabbR-Core/wwwroot/Chat.js

```diff
@@ -7,6 +7,9 @@
 (function ($, connection, window, ui, utility) {
     "use strict";
 
+    // Comment out to ensure logging is turned off.
```

Comment:
I'll remove for now, but it would be a good idea to have something like "if environment is development, add logging cookie", because when we run in incognito by default the cookie isn't present and needs to be re added every time.